test: Move files to the Browser folder

SamuelSalas opened the pull request:
<!--
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been completely filled out, and PR status checks have passed at least once.
-->

## **Description**

We currently have a few of our page object files lingering outside of a specific folder. As part of our ongoing efforts to improve the organization and maintainability of our test automation codebase, we need to move Page Object files into their respective folders. This will help streamline the project structure, making it easier for team members to locate and manage the files. The folder where each of the remaining files should be moved have to reflect the functionality or feature on the app. This same structure should be reflected on the selectors folder.



## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**

Regression test run https://app.bitrise.io/app/be69d4368ee7e86d/pipelines/46998a35-cba3-4462-a63d-398ababf4504?tab=workflows

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

SamuelSalas committed:
Move files to the Browser folder

SamuelSalas committed:
Merge branch 'main' into tes/2000-e2e-move-files-to-browser-folder

SamuelSalas committed:
fix lint issues

SamuelSalas committed:
Restore DrawerView

SamuelSalas committed:
Merge branch 'main' into tes/2000-e2e-move-files-to-browser-folder

SamuelSalas committed:
Merge branch 'main' into tes/2000-e2e-move-files-to-browser-folder

SamuelSalas committed:
Merge branch 'main' into tes/2000-e2e-move-files-to-browser-folder

cortisiko commented on e2e/pages/Browser/BrowserView.js:
```diff
@@ -1,42 +1,28 @@
 import TestHelpers from '../../helpers';
```
Because we are refining the add browser favorites workflow, i recommend **not** removing these methods as we will reuse them soon. So go ahead and readd them!

cortisiko commented on e2e/pages/Browser/AddBookmarkView.js:
```diff
@@ -12,10 +12,6 @@ class AddFavoritesView {
       : Matchers.getElementByLabel(AddBookmarkViewSelectorsIDs.CONFIRM_BUTTON);
   }
-
-  async tapAddBookmarksButton() {
```
Readd!

cortisiko reviewed (changes requested):
looks good, just a few comments

SamuelSalas committed:
Readd removed methods and selectors from AddBookmarkView and BrowserView page objects

SamuelSalas committed:
Merge branch 'main' into tes/2000-e2e-move-files-to-browser-folder

bfullam committed:
refactor: replace swap slider with swap button (#12030)

<!--
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been
completely filled out, and PR status checks have passed at least once.
-->

## **Description**

Feedback was received suggesting that the "swipe to swap" UI was often
navigating back instead of executing the swap. This PR replaces the
swipe with a button to address this issue and simplify the interaction.

<!--
Write a short description of the changes included in this pull request,
also include relevant motivation and context. Have in mind the following
questions:
1. What is the reason for the change?
2. What is the improvement/solution?
-->

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to the swap page
2. Get quotes
3. See swap button

## **Screenshots/Recordings**
<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**
<img width="310" alt="Screenshot 2024-10-25 at 17 22 47"
src="https://github.com/user-attachments/assets/f93ba955-164e-4a6b-895b-400a56c8bda0">
<!-- [screenshots/recordings] -->

### **After**
<img width="309" alt="Screenshot 2024-10-25 at 17 24 04"
src="https://github.com/user-attachments/assets/3aacf0c1-1066-4dbd-ac62-bef8682c2660">
<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Contributor
Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile
Coding
Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [x] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

cortisiko commented on e2e/pages/Browser/ConnectedAccountsModal.js:
```diff
@@ -1,7 +1,7 @@
 import {
```
Back when we first created the POM, we had a bunch of modals but as time went by we converted these modals into bottom sheets. With that said, Rename file to ConnectedAccountBottomSheet

cortisiko commented on e2e/pages/Browser/ContractApprovalModal.js:
```diff
@@ -1,7 +1,7 @@
 import {
```
Rename the file to ContractApprovalBottomSheet

cortisiko commented on e2e/pages/Browser/SpamFilterModal.js:
```diff
@@ -1,6 +1,6 @@
 import Matchers from '../../utils/Matchers';
```
Rename file to SpamFilterBottomSheet

cortisiko commented on e2e/pages/Browser/SigningModal.js:
```diff
@@ -1,4 +1,4 @@
-import { SigningModalSelectorsIDs } from '../../selectors/Modals/SigningModal.selectors';
```
Rename to SigningBottomSheet

cortisiko commented on e2e/pages/Browser/ConnectModal.js:
```diff
@@ -1,7 +1,7 @@
 import {
```
Rename file to ConnectAccountBottomSheet

cortisiko committed:
test: Move portfolio connect out of quarantine (#12023)

<!--
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been
completely filled out, and PR status checks have passed at least once.
-->

The Purpose of this task is to readd the portfolio connect e2e to the
regression tag.

Fixes:

1. Go to this page...
2.
3.

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

<!-- [screenshots/recordings] -->

<!-- [screenshots/recordings] -->

- [ ] I’ve followed [MetaMask Contributor
Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile
Coding
Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

Co-authored-by: Aslau Mario-Daniel <marioaslau@gmail.com>

SamuelSalas committed:
Change modal to BottomSheet

SamuelSalas committed:
Update selectors of bottomsheet

SamuelSalas committed:
Merge branch 'main' into tes/2000-e2e-move-files-to-browser-folder

SamuelSalas committed:
Update selectors of bottomsheet

SamuelSalas committed:
Merge branch 'main' into tes/2000-e2e-move-files-to-browser-folder

SamuelSalas committed:
Merge branch 'main' into tes/2000-e2e-move-files-to-browser-folder

SamuelSalas committed:
Change SigningModalSelectorsIDs for SigningBottomSheetSelectorsIDs

SamuelSalas committed:
Merge branch 'main' into tes/2000-e2e-move-files-to-browser-folder
